sim/priest/shadow_effects.go

```diff
@@ -4,6 +4,15 @@ import (
 	"github.com/wowsims/tbc/sim/core"
 )
 
+func (priest *Priest) applyTalentsToHolySpell(cast *core.Cast, effect *core.SpellHitEffect) {
+	effect.ThreatMultiplier *= 1 - 0.04*float64(priest.Talents.SilentResolve)
+	if cast.ActionID.SpellID == SpellIDSmite {
+		effect.BonusSpellCritRating += float64(priest.Talents.HolySpecialization) * 1 * core.SpellCritRatingPerCritChance
+	}
+
+	effect.BonusSpellCritRating += float64(priest.Talents.ForceOfWill) * 1 * core.SpellCritRatingPerCritChance
+}
+
```

Comment:
We can put this in talents.go too. In fact, if you want you can move both of the functions in this file into talents.go, and then delete this file.

Reply:
I agree moving both since its specific to talents, but there is more in the shadow_effects.go file I'll keep there for less clutter in talents.go.